run_fontbakery.sh

```diff
@@ -0,0 +1,14 @@
+mkdir -p build/fontbakery/
```

Comment:
```suggestion
#!/bin/sh

set -e  # make sure to abort on error
set -x  # echo commands

mkdir -p build/fontbakery/
```

And please run `chmod +x run_fontbakery.sh` and commit that change. Without this we can't run it as `./run_fontbakery.sh`